<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCSysLog</class>
 <widget class="QFrame" name="NewCSysLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1390</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统日志</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(D:/WinParkUI/debug/Image/NewIcon/CommonBigBG-normal.jpg);</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>1301</width>
     <height>741</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1281</width>
      <height>671</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="title">
     <string>操作种类</string>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>1261</width>
       <height>361</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>换班记录</string>
      </attribute>
      <widget class="QTableWidget" name="tableShift">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>1231</width>
         <height>351</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image:url(none);</string>
       </property>
       <row>
        <property name="text">
         <string>1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>2</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>操作者</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>内容</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>记录时间</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>卡片管理</string>
      </attribute>
      <widget class="QTableWidget" name="tableCardMgm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>1231</width>
         <height>351</height>
        </rect>
       </property>
       <row>
        <property name="text">
         <string>1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>2</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>操作者</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>内容</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>记录时间</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>删除数据库</string>
      </attribute>
      <widget class="QTableWidget" name="tablDeleteData">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>1231</width>
         <height>351</height>
        </rect>
       </property>
       <row>
        <property name="text">
         <string>1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>2</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>操作者</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>内容</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>记录时间</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>手动开关闸</string>
      </attribute>
      <widget class="QTableWidget" name="tableManualGate">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>1231</width>
         <height>351</height>
        </rect>
       </property>
       <row>
        <property name="text">
         <string>1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>2</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>操作者</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>内容</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>记录时间</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>硬件警告信息</string>
      </attribute>
      <widget class="QTableWidget" name="tableHardware">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>1231</width>
         <height>351</height>
        </rect>
       </property>
       <row>
        <property name="text">
         <string>1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>2</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>操作者</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>内容</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>记录时间</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>1261</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <property name="title">
      <string>手动开闸抓拍</string>
     </property>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>241</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(none);</string>
      </property>
      <property name="title">
       <string>入口</string>
      </property>
      <widget class="QLabel" name="lblImg0">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>15</y>
         <width>231</width>
         <height>201</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>241</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(none);</string>
      </property>
      <property name="title">
       <string>出口</string>
      </property>
      <widget class="QLabel" name="lblImg1">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>15</y>
         <width>231</width>
         <height>201</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>20</y>
        <width>241</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(none);</string>
      </property>
      <property name="title">
       <string>库中库入口</string>
      </property>
      <widget class="QLabel" name="lblImg2">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>15</y>
         <width>231</width>
         <height>201</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>20</y>
        <width>241</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(none);</string>
      </property>
      <property name="title">
       <string>库中库出口</string>
      </property>
      <widget class="QLabel" name="lblImg3">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>15</y>
         <width>231</width>
         <height>201</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>20</y>
       <width>497</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>开始时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dtStart">
        <property name="displayFormat">
         <string>yyyy-MM-dd HH:mm:ss</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>截止时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dtEnd">
        <property name="displayFormat">
         <string>yyyy-MM-dd HH:mm:ss</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnQuery">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnMinimalize">
    <property name="geometry">
     <rect>
      <x>613</x>
      <y>710</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>5</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);
color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblClose">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>6</y>
     <width>30</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
